ext/hash/sha3/generic32lc/brg_endian.h

```diff
@@ -44,9 +44,7 @@
 #elif defined( __linux__ ) || defined( __GNUC__ ) || defined( __GNU_LIBRARY__ )
 #  if !defined( __MINGW32__ ) && !defined( _AIX )
 #    include <endian.h>
-#    if !defined( __BEOS__ )
-#      include <byteswap.h>
-#    endif
+#    include <byteswap.h>
```

Comment:
Here might be even useful to remove the entire `if 0` block with not used code here... I don't know if these are useful as comments but certainly look a bit bloated when reading the code.